app/models/ebean/Alert.java

```diff
@@ -0,0 +1,201 @@
+/**
+ * Copyright 2015 Groupon.com
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package models.ebean;
+
+import com.avaje.ebean.Model;
+import com.avaje.ebean.annotation.CreatedTimestamp;
+import com.avaje.ebean.annotation.UpdatedTimestamp;
+
+import java.sql.Timestamp;
+import java.util.UUID;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.persistence.Version;
+
+/**
+ * Data model for alerts.
+ *
+ * @author Deepika Misra (deepika at groupon dot com)
+ */
+@Entity
+@Table(name = "alerts", schema = "portal")
+public class Alert extends Model {
+
+    @Id
```

Comment:
I dont think it makes a difference either way here since the ID actually comes from the alert model (the internal one) and we are not relying on the DB to generate an id. I think its ok to do this here because the UUID is an attribute on the model here and not just a unique identifier for database sake (like in case of Host). Thoughts?